README.md

```diff
@@ -92,13 +92,23 @@ Two specific points to expand on:
 
 ### Setting up
 
+Running the app requires only a standard Flutter setup,
+using the Flutter `main` channel:
+
 1. Follow the [Flutter installation guide](https://docs.flutter.dev/get-started/install)
    for your platform of choice.
 2. Switch to the latest version of Flutter by running `flutter channel main`
    and `flutter upgrade` (see [Flutter version](#flutter-version) below).
 3. Ensure Flutter is correctly configured by running `flutter doctor`.
 4. Start the app with `flutter run`, or from your IDE.
 
+Parts of our test suite require an additional dependency:
+
+5. Install SQLite, for example by running `sudo apt install libsqlite3-dev`.
```

Comment:
@Khader-1 The previous draft called for installing `sqlite3` too. I *think* that shouldn't be needed, though I'm not certain of that — that package is mainly just the `sqlite3` command-line binary:
```
$ dpkg -L sqlite3
/.
/usr
/usr/bin
/usr/bin/sqldiff
/usr/bin/sqlite3
/usr/share
/usr/share/doc
/usr/share/doc/sqlite3
/usr/share/doc/sqlite3/changelog.Debian.gz
/usr/share/doc/sqlite3/changelog.gz
/usr/share/doc/sqlite3/changelog.html.gz
/usr/share/doc/sqlite3/copyright
/usr/share/man
/usr/share/man/man1
/usr/share/man/man1/sqlite3.1.gz
```

(So specifically it's just that, and the `sqldiff` command-line tool which I hadn't heard of until just now, and some documentation. At least that's what it is on my system.)